requirements.txt

```diff
@@ -10,18 +9,19 @@ h5py<3.0.0
 jsonschema
 moto==1.3.14
 numba==0.53.1
-numpy==1.21.2
+numpy==1.22.2
```

Comment:
This breaks numba, but we don't seem to _use_ numba in portal anymore

Reply:
In that case should we just remove the req?

Reply:
broke upload pipeline, reverted